operating_unit/view/operating_unit_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_operating_unit_form" model="ir.ui.view">
+            <field name="name">operating.unit.form</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <form string="Operating Unit" version="7.0">
+                    <group>
+                        <field name="name"/>
+                        <field name="code"/>
+                        <field name="active"/>
+                        <field name="partner_id"/>
+                        <field name="company_id" groups="base.group_multi_company"/>
+                    </group>
+                </form>
+            </field>
+        </record>
+
+        <record id="view_operating_unit_tree" model="ir.ui.view">
+            <field name="name">operating.unit.tree</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <tree string="Operating Unit">
+                    <field name="code"/>
+                    <field name="name"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                </tree>
+            </field>
+        </record>
+        <record id="view_operating_unit_search" model="ir.ui.view">
+            <field name="name">operating.unit.search</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <search string="Search Operating Unit">
+                    <field name="name" string="Operating Unit"/>
+                    <field name="code" string="Code"/>
+                </search>
+            </field>
+        </record>
+
+        <record id="action_operating_unit_tree" model="ir.actions.act_window">
+            <field name="name">Operating Units</field>
+            <field name="res_model">operating.unit</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to start a new Operating Unit.
+              </p>
+            </field>
+        </record>
+
+       <menuitem action="action_operating_unit_tree"
+                 id="manu_action_operating_unit_tree"
+                 parent="base.menu_users"
+                 groups="group_multi_operating_unit"/>
```

Comment:
even if it's the group is defined in this module prefixing the group xml_id by the module name isn't a good practice?